src/endpoint.js

```diff
@@ -74,6 +76,19 @@ module.exports = class AbstractEndpoint {
     return this
   }
 
+  //turns on auto-batching for this endpoint
+  enableAutoBatch (interval) {
+    if (this._autoBatch === null) {
+      this._autoBatch = {
+        interval: interval || 1000,
```

Comment:
- Maybe rename into `windowSize`? Interval means more that it runs in that specific interval even it if didnt get called.
- This feels like a pretty high default to me, since it adds 1s to each batched request (that takes <200ms usually). 50ms or something like that seems more reasonable.

Reply:
Updated the name to batchDelay, and had updated the delay to 100 before I caught this comment, can reduce to 50 if that makes more sense to you

Reply:
I feel like 50 is probably the better default, just because 100 is already 50% of just shooting out the request instantly